tubular/edx_api.py

```diff
@@ -88,17 +113,19 @@ def _giveup_on_unexpected_exception(exc):
     """
     Giveup method that gives up backoff upon any unexpected exception.
     """
-    return not ( (500 <= exc.response.status_code < 600 and exc.response.status_code != 504) or exc.response.status_code == 104)
+    return not ((
+                        500 <= exc.response.status_code < 600 and exc.response.status_code != 504) or exc.response.status_code == 104)
 
 
 def _retry_lms_api():
     """
     Decorator which enables retries with sane backoff defaults for LMS APIs.
     """
+
```

Comment:
I don't like blank lines after the docstrings))
And [PEP says the same)](https://peps.python.org/pep-0257/#:~:text=There%E2%80%99s%20no%20blank%20line%20either%20before%20or%20after%20the%20docstring)

Reply:
done